src/planning.py

```diff
@@ -51,6 +51,7 @@ def sesame_plan(
     max_horizon: int,
     check_dr_reachable: bool = True,
     allow_noops: bool = False,
+    abstract_ldl: LiftedDecisionList = None,
```

Comment:
let's change this back to an abstract policy, since there could be other kinds of policies that we would use here instead. inside of the PG4 approach, you can write something like:
```
abstract_policy = lambda a,o,g: utils.query_ldl(self._current_ldl, a, o, g)
```
and then inside of sesame, you can just use `abstract_policy(atoms, objects, goal)`